backend/src/main/java/gov/cdc/usds/simplereport/idp/authentication/DemoOktaAuthentication.java

```diff
@@ -86,9 +130,18 @@ public void reset() {
     @Getter @Setter private String password;
     @Getter @Setter private String recoveryQuestion;
     @Getter @Setter private String recoveryAnswer;
+    @Getter @Setter private DemoMfa mfa;
```

Comment:
Do we need to support multiple factors?  I'm guessing our initial workflow will only support adding one so we don't need to be able to test multiple.

Reply:
I think eventually we'll need to be able to support multiple factors, but for now it should only be possible from the frontend to enroll in a single factor. 
So, to answer your question: no, we don't need need to be able to test multiple (yet). :)